### ISSUE TITLE
sticky backward seeking and undraggable segment handles

### ISSUE THREAD

**Issue by un-chein-andalou** [bbcrd/peaks.js#105]
Hi folks,

I just started playing with peaks and find it's mostly working brilliantly! but a couple of things..

 I don't know why but backward seeking is kinda sticky:
When clicking far ahead of the playhead in the overview view the zoom view jumps right to the correct spot -awesome, but when clicking far behind the play head in the overview view the zoom view doesn't move all the way to the correct spot -less-awesome. With a big sized audio file you have to do a lot of clicking on the overview to move the zoom view all the way back so this can be annoying.

also I wonder why dragging segments is disabled in the overview:
in waveform.segments.js ln 68:70
```javascript
          if (segmentGroup === segmentOverviewGroup) {
            draggable = false;
          }
```
I guess dragging segments in the overview was disabled for a reason, but still I need to drag segment handles around in the overview so I went ahead and enabled it. Will it cause problems? I haven't seen any issues so far but thought I'd check.

**Comment by oncletom** [comment]
Hello @un-chein-andalou, sorry for the delay and thank you for sharing your findings. Any chance you can propose that as a PR? Thanks :-)

**Comment by un-chein-andalou** [comment]
created PR #108

**Comment by oncletom** [comment]
Thanks @un-chein-andalou, I will try to give it a go next week :-)

**Comment by oncletom** [comment]
Solved by #108
